### ISSUE TITLE
phlay a commit with an empty file gives an obscure error

### ISSUE THREAD

**Issue by Rob--W** [mystor/phlay#58]
I just tried to upload a revision to Phabricator, but it failed with the following obscure error:

```
$ phlay HEAD

4bd448e77c1c Bug 1544172 - Separate schemas for native messaging APIs r?kmag
  New Revision
    mozilla-central
  Bug 1544172
    Separate schemas for native messaging
Traceback (most recent call last):
  File "/path/to/bin/phlay", line 998, in <module>
    main()
  File "/path/to/bin/phlay", line 984, in main
    process_args(args)
  File "/path/to/bin/phlay", line 835, in process_args
    diff = commit.diff()
  File "/path/to/bin/phlay", line 629, in diff
    self._diff = Diff(self)
  File "/path/to/bin/phlay", line 248, in __init__
    self.parse_change(c)
  File "/path/to/bin/phlay", line 314, in parse_change
    ))[2:]
ValueError: not enough values to unpack (expected at least 1, got 0)
```

After some debugging I found that one of the files in my commit is empty.
After removing the file and amending the commit, the upload succeeded.

**Comment by mystor** [comment]
Huh, that's super weird. I thought I had fixed this in #49 

Maybe this is another weird situation where the file only has a newline or something, which could throw off the object ID?

**Comment by Rob--W** [comment]
I probably experienced this bug since I was using Phlay 0.1.4.
I didn't know that I was using an old version, but I will submit a pull request that fixes the update check.
